app/templates/components/recruiter-conversations/conversation.hbs

```diff
@@ -11,15 +11,14 @@
   {{/card.title}}
 
   <div class="md-actions" layout-align="center">
-    <a target="_blank" href="{{conversation.engineer.website~}}">
-      {{#paper-button}}View their website{{/paper-button}}
-    </a>
+    {{!-- inconsistent syntax due to a problem with using a paper-button with an href attribute
+    filed an issue on the ember-paper github --}}
+    <a target="_blank" href="{{conversation.engineer.website}}">{{#paper-button}}View their website{{/paper-button}}</a>
+
     {{#link-to 'recruiter.view-conversation' conversation.id ~}}
       {{#paper-button}}View invitation{{/paper-button}}
     {{~/link-to}}
-    {{#paper-button~}}
-      <div {{action 'deleteConvo'}}> Delete </div>
-    {{~/paper-button}}
+    {{#paper-button onClick=(action "deleteConvo")}}Delete{{/paper-button}}
 
   </div>
 {{/paper-card}}
```

Comment:
Link to issue on ember-paper GitHub https://github.com/miguelcobain/ember-paper/issues/748